geopandas/base.py

```diff
@@ -88,13 +88,23 @@ def _delegate_property(op, this):
         return Series(data, index=this.index)
 
 
-def _delegate_geo_method(op, this, *args, **kwargs):
+def _delegate_geo_method(op, this, **kwargs):
```

Comment:
Restricting this to `**kwargs` only ensures we always have a key to be printed in the error message.